packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.Props.ts

```diff
@@ -20,6 +20,9 @@ export interface IBreadcrumbProps extends React.Props<Breadcrumb> {
    * If not specified, all breadcrumbs will be rendered.
    */
   maxDisplayedItems?: number;
+
+  /** Method to call when trying to render an item. */
+  onRenderItem?: (item?: IBreadcrumbItem) => React.ReactNode;
```

Comment:
You want to use `IRenderFunction<IBreadcrumItem>` here, as @dzearing suggested in the issue.